### ISSUE TITLE
G65 requires not linking current link in breadcrumb; overly prescriptive?

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#4024]
[Technique G65](https://www.w3.org/WAI/WCAG22/Techniques/general/G65) seems to imply in its tests that it is essential to not link to the current page in a breadcrumb navigation to sufficiently meet [SC 2.4.8 Location](https://www.w3.org/WAI/WCAG22/Understanding/location):

> 5. For a breadcrumb trail that does include the current location:
>   - Check that all elements except for the current location are implemented as links.
>   - Check that the current location is not implemented as a link.

Considering that SC 2.4.8 only requires that …

> Information about the user's location within a set of Web pages is available.

… I cannot see how including or excluding a link to the current page is relevant to passing or failing the SC.

If my assumption is correct, I think the page should be adjusted to remove the step from the testing procedure. Furthermore, I think with the introduction of `aria-current="true"`, both including the link to the current page and leaving it off are equally good options.

**Comment by patrickhlauke** [comment]
Duplicate of https://github.com/w3c/wcag/issues/3948

Will be addressed once https://github.com/w3c/wcag/pull/3951 is merged

**Comment by yatil** [comment]
Thanks @patrickhlauke! I was under the impression I had searched for an open issue, but apparently not :-)
